hackyourrepo-app/AmirHossein/util/fetchData.js

```diff
@@ -0,0 +1,8 @@
+
+// const errorBox = elFactory('div', {id: 'error'});
+async function fetchData(url) {
```

Comment:
why don't you use this function in the other examples or wasn't it not part of the homework?